code/game/dna/genes/monkey.dm

```diff
@@ -37,8 +37,8 @@
 
 
 	var/mob/living/carbon/human/monkey/O = null
-	if(H.species.primitive)
-		O = new H.species.primitive(src)
+	if(H.species.primitive_form)

```

Comment:
There is no variable named primitive in the species datum. My guess is that it was renamed when species were changed to a new system, and this file wasn't updated.